lib/bloom.js

```diff
@@ -2,20 +2,6 @@ const assert = require('assert')
 const utils = require('ethereumjs-util')
 const byteSize = 256
 
-/**
- * Drops leading zeroes from a buffer
- * @function dropLeadingZeroes
- * @param {Buffer} buff
- * @returns {Buffer} a slice of the given buffer starting at the first non-zero entry
- */
-function dropLeadingZeroes (buff) {
```

Comment:
This was introduced fairly recently (< 1 year ago). Can you please look up in `git blame` which PR introduced it and why? There was some reasoning.

Reply:
Never mind, you've introduced it, nevertheless having the PR number linked here helps future tracking why things change.

Reply:
PR was #295

Reply:
As mentioned in my original comment I've only reverted this to demonstrate the scope of test failures this change has caused. This change has also caused other failures to be masked which raises the question as to the correct solution?

Reply:
Hi @AusIV, the changes introduced in https://github.com/ethereumjs/ethereumjs-vm/pull/295 resulted in a somewhat conflicting situation regarding passing of blockchain tests, see the thread in this issue. Have you got some deeper insight on this and can eventually join the discussion?